gsb_research_hub_subtheme.theme

```diff
@@ -109,3 +109,61 @@ function gsb_research_hub_subtheme_preprocess_field(&$variables) {
     $su_paragraph_row__bgcolor = $variables['items'][0]['content']['#markup'];
   }
 }
+
+// Alter the results of the people page.
+function gsb_research_hub_subtheme_views_pre_render(&$view) {
```

Comment:
You're not expecting multiple pages of results here correct? I'm asking because if you get into a pager scenario, messing with sorting after the DB query will bring you much fun.